floodsub.go

```diff
@@ -324,6 +370,75 @@ func msgID(pmsg *pb.Message) string {
 	return string(pmsg.GetFrom()) + string(pmsg.GetSeqno())
 }
 
+// pushMsg pushes a message performing validation as necessary
+func (p *PubSub) pushMsg(vals []*topicVal, src peer.ID, msg *Message) {
+	if len(vals) > 0 {
+		// validation is asynchronous and globally throttled with the throttleValidate semaphore.
```

Comment:
Is this comment still true?

Reply:
Yes it is still true. 
I can remove the global throttle if we deem so, I just like to have a limit on how much goncurrency we can spawn as a result of incoming network traffic.

Reply:
although, subscription reference below! that's not true, I will fix the comment.